client/rest/src/main/java/org/opensearch/client/RestClient.java

```diff
@@ -131,6 +132,7 @@ public class RestClient implements Closeable {
     private volatile NodeTuple<List<Node>> nodeTuple;
     private final WarningsHandler warningsHandler;
     private final boolean compressionEnabled;
+    private final boolean chunkedTransferEncodingEnabled;
```

Comment:
Just a suggestion, may be shorter name like `chunkedEnabled` would be more convenient? (and aligned with `HttpEntity` as well)?